azurerm_api_management/main.tf

```diff
@@ -14,14 +14,14 @@ data "azurerm_key_vault_secret" "certificate_secret" {
 }
 
 module "secrets_from_keyvault" {
-  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_secrets_from_keyvault?ref=v2.0.5"
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_secrets_from_keyvault?ref=v2.0.25"
```

Comment:
Release v2.0.25 does not exist yet, but it's required. It refers to at least itself